packages/core/src/components/StepList/StepList.scss

```diff
@@ -0,0 +1,356 @@
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Step List
+
+@status beta
+
+A step list represents a user's progression through an application or multi-page
+form. It serves as a table of contents and a way to quickly see where they are
+and what they should be working on next.
+
+Markup:
+<div style="max-width: 628px">
+  <ol class="ds-c-step-list ds-u-margin-top--4">
+    <li class="ds-c-step ds-c-step--completed">
+      <h2 class="ds-c-step__header">
+        <div class="ds-c-step__title">Choose a tax year</div>
+        <div class="ds-c-step__actions">
+          <div class="ds-c-step__completed-text">Completed</div>
+          <a href="#step-1">
+            Edit<span class="ds-u-visibility--screen-reader">
+              {" "}
+              "Choose a tax year"
+            </span>
+          </a>
+        </div>
+      </h2>
+    </li>
+    <li class="ds-c-step ds-c-step--current">
+      <h2 class="ds-c-step__header">
+        <div class="ds-c-step__title">Enter household details</div>
+        <div class="ds-c-step__actions">
+          <a href="#step-2" class="ds-c-button ds-c-button--primary">
+            Resume<span class="ds-u-visibility--screen-reader">
+              {" "}
+              "Enter household details"
+            </span>
+          </a>
+        </div>
+      </h2>
+      <div class="ds-c-step__description">
+        Answer questions about who in your household qualifies for a premium tax
+        credit and information on each person, including date of birth,
+        location(s) they lived in for the year, and months of marketplace
+        coverage.
+      </div>
+      <ol class="ds-c-step__substeps">
+        <li class="ds-c-substep">
+          <div class="ds-c-substep__line">
+            <div class="ds-c-substep__title">Overall household</div>
+            <a href="#step-2a">
+              Edit<span class="ds-u-visibility--screen-reader">
+                {" "}
+                "Overall household"
+              </span>
+            </a>
+          </div>
+        </li>
+        <li class="ds-c-substep">
+          <div class="ds-c-substep__line">
+            <div class="ds-c-substep__title">Bob's information</div>
+          </div>
+        </li>
+        <li class="ds-c-substep">
+          <div class="ds-c-substep__line">
+            <div class="ds-c-substep__title">Barb's information</div>
+          </div>
+        </li>
+      </ol>
+    </li>
+    <li class="ds-c-step">
+      <h2 class="ds-c-step__header">
+        <div class="ds-c-step__title">Review your information</div>
+        <div class="ds-c-step__actions" />
+      </h2>
+    </li>
+    <li class="ds-c-step">
+      <h2 class="ds-c-step__header">
+        <div class="ds-c-step__title">View premium results</div>
+        <div class="ds-c-step__actions" />
+      </h2>
+    </li>
+  </ol>
+</div>
+
+Style guide: components.step-list
```

Comment:
Cross posting from Slack...

![image](https://user-images.githubusercontent.com/371943/33496919-ff6d8928-d699-11e7-8405-54f718c63709.png)

This would mean swapping `components` with `patterns` here.